### ISSUE TITLE
FR - Recordings - Grouping Setting

### ISSUE THREAD

**Issue by spaceboy76** [EmbySupport/Emby.WMC.UI#12]
Within the live tv recordings screen of WMC there was a way to flick instantly between an episode timeline and a timeline grouped by series, whereby if you entered the series "folder", all series epsiodes were displayed in a timeline. There was some indicator of unwatched episodes in a series.

its no different really than the two views within emby:
- live tv > Recordings is the episode timeline and
- Recorded shows is the series timeline

but having them accessible from one screen and just being able to flick instantly between the views was so much more convenient than emby. it sounds trivial but it was the first thing i looked for in the beta - thats how much i miss it 10 years later!

the one thing i don't remember was how the the switch was initiated - i'm pretty certain it was a button on the remote rather than on the OSD though

**Comment by softworkz** [comment]
What do you mean by "flick"? 

The Recorded TV View in WMC looks like this:

![image](https://github.com/EmbySupport/Emby.WMC.UI/assets/4985349/e15b5512-f9e5-4bfa-baeb-08002098b3a5)

**Comment by spaceboy76** [comment]
yep thats it. the "date recorded" menu is what i mean by the episode timeline and if you change to the "title" menu then i believe you get a series timeline view.

so in this context, by flick i meant changing between those two sub menus

**Comment by softworkz** [comment]
Don't we have almost the same currently in the WMC UI?

The "recordings" tab shows items chronologically and the "series" tab by series?

![image](https://github.com/EmbySupport/Emby.WMC.UI/assets/4985349/34f388c5-9f9f-4c96-883f-0485e317e8e7)

**Comment by spaceboy76** [comment]
no. assuming series is the same as the series tab in emby, then this is the view of series recording settings. nothing to do with recordings already made.

also, the series recording settings don't actually show after selecting a series

**Comment by softworkz** [comment]
I'm off-site. I'll get back in a few days. Thanks.

**Comment by softworkz** [comment]
I had discussed this with Luke. It is currently not (easily) possible to achieve this.
There are plans for making this possible, but this is functionality which needs to be added to the server(!) and the core client first.

Could you perhaps create a corresponding feature request for the core client?

Thanks

**Comment by spaceboy76** [comment]
Thanks, but do you really think a feature request is necessary? Emby already has this view, as indicated. i personally believe its more valuable than the episode timeline anyway - with the episode timeline its not easy to tell which episode from a series is next up, the only way is to go back and search by eye, which isnt a great user experience.

**Comment by softworkz** [comment]
Then I might have misunderstood. Can you please post a screenshot of the Emby view you are referring to?

**Comment by spaceboy76** [comment]
sure - this view (with sorting)
![image](https://github.com/EmbySupport/Emby.WMC.UI/assets/9905053/ec63d769-d5b4-42cf-9253-6a361eb29dcb)
is identical to the "recorded tv -> title" view in wmc, except for 2 minor things:

- emby allows you to set different folders for recorded series and recorded movies - separate libraries are created for each. in wmc these libraries would be combined to give one overall view of all recorded tv
- in wmc there was just a single row of thumbnails - thumbnails were screen grabs as you have them in your recorded tv view. i'm not really bothered about looks, as long as i can see an unwatched indicator i'm happy either way

**Comment by softworkz** [comment]
Sorry for being unclear:

> Emby already has this view, as indicated

I meant the view you mentioned which Emby would have already..

**Comment by spaceboy76** [comment]
yes, i agree. we are still talking at cross purposes. 

what i want is for emby WMC to mirror the 2 main options/views available in WMC recorded tv. From the first pic in this thread:

1. date recorded - shows a timeline of recordings most recent first. this is the current view in all emby apps for live tv -> recordings. big tick here, no problems. this is shown in the second pic in this thread.
2. title - shows a timeline of recordings, most recent first BUT recordings are grouped by series. emby has this view as shown in the third pic in this thread.

the problem is that in WMC, i can simply "flick" between these two different views from one screen with one or two presses of a button. In emby i have to exit live tv recordings (equivalent to view 1), 3 button presses - one to return to the top of the recordings list, one to get to the menu at the top of the screen and a further one to finally get back to the home screen. i then need to go and find the recorded shows library, which i have relegated to the bottom of my list of libraries because to be honest, even though i think this view is superior, it is a duplication of live tv -> recordings, probably another 20+ clicks. Finally i get to something equivalent to view 2 Title.

As i intimate, i don't really understand why Emby (or even WMC tbh) prioritise view 1 over view 2. There is huge advantage to view 2 and zero drawbacks. but the facts are they do, and any feature request to replace view 1 with view 2 will be a "nice idea for the future" with less likelihood of materialising than those mexican aliens. 

however as emby has the concept of both views AND you are creating a replica of WMC, i am hoping you can put a shortcut/menu to view 2 directly in live tv. Thereby replicating the ability to flick between the two different views.

to me this falls under your third rule for requesting new features "Feature requests requiring extra-coding are okay when it's about doing something in a WMC way".

if not, then i guess i am back to hoping for those mexican aliens to appear...

cheers

**Comment by softworkz** [comment]
Thanks a lot for the detailed explanation!

It turns out that I had actually gotten you right when I answered this:

> I had discussed this with Luke. It is currently not (easily) possible to achieve this. There are plans for making this possible, but this is functionality which needs to be added to the server(!) and the core client first.

I guess I should have given a few more details on the subject. The problem is that we are dealing with two different kinds of items, like apples and oranges:

- A: What you see on the "Recorded TV" >> "Series" tab are "Series Timers" - which are representing instructions to record something
- B: What we have in the "Recorded Shows" library(!) are actual tv shows like in other Emby libraries as well.

The problem is that there don't exist any relations between A and B (even the series and/or episode names can mismatch).

That's what I was asking Luke: whether it's possible in some way to relate them. The answer is "no, not currently".

> to me this falls under your third rule for requesting new features "Feature requests requiring extra-coding are okay when it's about doing something in a WMC way"

Yes, it normally would, but in this case, a conceptual change is required which also requires changes at the server side, which is why it falls out of this category. It's not achievable by a pure client side change.

That's why I was asking for this:

> Could you perhaps create a corresponding feature request for the core client?

Because a user request has more weight than anything needed for the WMC UI (capacity for this is exhausted, I think 😆🤣😆🤣)
